tests/stability_test.go

```diff
@@ -2,33 +2,22 @@ package tests_test
 
 import (
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 
-	"kubevirt.io/kubevirt/tests/util"
-
-	"kubevirt.io/client-go/kubecli"
 	"kubevirt.io/kubevirt/tests"
-	cd "kubevirt.io/kubevirt/tests/containerdisk"
+	"kubevirt.io/kubevirt/tests/libvmi"
 )
 
+// Replace PDescribe with FDescribe in order to measure if your changes made
+// VMI startup any worse
```

Comment:
I don't think this setup makes much sense honestly; this test is not being executed - `PDescribe` . 

Shouldn't it be **always** executed by CI - even if on a separate, scale/performance scoped - lanes ?

I don't think we can expect people to use this ... Honestly, I have never executed this in my life (wasn't aware it even existed ...).

Anyway, this is clearly off-scope of your PR...

Regarding the comment you've introduced, I am unsure it makes sense to keep it in the code...

... BUT it proved extremely valuable to raise this (hypothetical..) issue. I would have probably missed the `PDescribe` stanza without the comment.

Reply:
Have you been reading my mind? That was precisely my thought process. I did not understand why this test does not run and why it exists. I did not even know what `PDescribe` earlier today...

The comment is there exactly because I suspect readers to have the same questions. This is not a CI test, but a developer helper tool. Should it remain or deleted? The purpose of this PR (and of code cleanup in general) is to make these questions apparent. A following PR can kill this test.

Reply:
I added it roughly 3.5 years ago where we did not have the capacity and stability to run this permanently on CI. I forgot about it. Now with the sig-performance work, we may have the place to enable it, or remove it and replace it with other things.